cmd/kpod/logs.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	logsFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:   "details",
+			Usage:  "Show extra details provided to the logs",
+			Hidden: true,
+		},
+		cli.BoolFlag{
+			Name:  "follow, f",
+			Usage: "Follow log output.  The default is false",
+		},
+		cli.StringFlag{
+			Name:  "since",
+			Usage: "show logs since TIMESTAMP",
+		},
+		cli.StringFlag{
+			Name:  "tail",
+			Usage: "output the specified number of LINES at the end of the logs.  Defaults to 'all'",
+		},
```

Comment:
nit of a nit.  Some of these usage statements start with an uppercase, others with lowercase.  I'd prefer upper, either is fine, just be consistent plz.